### ISSUE TITLE
[BUG] Invalid URL for tkn

### ISSUE THREAD

**Issue by pcarney8** [redhat-actions/openshift-tools-installer#57]
### Version
`redhat-actions/openshift-tools-installer@v1`

### Describe the bug
`Error: Invalid URL: /pub/openshift-v4/x86_64/clients/pipeline/0.13.1/`
it appears the `openshift-tools-installer` has an invalid URL to where the tools are. it appears https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.13.1/ shows the tarballs but i'm not seeing an `x86_64` in the path

### Steps to reproduce, workflow links, screenshots

```
Run redhat-actions/openshift-tools-installer@v1
Installing tkn matching version "0.13.1"
ℹ️ Tools will be installed from "mirror".

🔎 Searching for a version of tkn satisfying the range "0.13.1" that was input as "0.13.1"
Download directory for tkn is https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/
Max tkn version satisfying 0.13.1 is 0.13.1
Error: Invalid URL: /pub/openshift-v4/x86_64/clients/pipeline/0.13.1/

❌ Failed to install tkn.

Using temporary directory for storage
📁 CLIs will be downloaded to /home/runner/work/_temp/openshift-bin
Added /home/runner/work/_temp/openshift-bin to PATH
Wrote out installed versions to /home/runner/work/_temp/openshift-bin/openshift-clients-installed.json
```

**Comment by divyansh42** [comment]
Thanks for reporting this. We are investigating this problem and update you as soon as possible.
If your use case satisfies please feel free to use input `source: github` to download and install binaries from GitHub instead of OpenShift Mirror.

**Comment by divyansh42** [comment]
Related issue https://github.com/redhat-actions/openshift-tools-installer/issues/58

**Comment by divyansh42** [comment]
@pcarney8 We have fixed this issue, could you please re-run your workflows and confirm.

**Comment by divyansh42** [comment]
Resolved in #59
Feel free to re-open if you find this not working.
